microsoft-edge/index.yml

```diff
@@ -0,0 +1,445 @@
+        - url: ./extensions-chromium/getting-started/part2-content-scripts.md
+          itemType: reference
+          text: Create a simple extension – part 2
+    # Card
+    - title: "Extensions: Developer Guide"
+      links:
+        - url: ./extensions-chromium/developer-guide/port-chrome-extension.md
+          itemType: reference
+          text: Port Chromium extension to Microsoft Edge
+        - url: ./extensions-chromium/developer-guide/alternate-distribution-options.md
+          itemType: reference
+          text: Alternate methods of distributing extensions
+        - url: ./extensions-chromium/developer-guide/native-messaging.md
+          itemType: reference
+          text: Native messaging
+    # Card
+    - title: "Extensions: Publish to the Microsoft Store"
+      links:
+        - url: ./extensions-chromium/publish/publish-extension.md
+          itemType: reference
+          text: Step-wise publishing process
+        - url: ./extensions-chromium/publish/create-dev-account.md
+          itemType: reference
+          text: Open developer account
+        - url: ./extensions-chromium/publish/upload-video.md
+          itemType: reference
+          text: Upload YouTube Video
+        - url: ./extensions-chromium/publish/update.md
+          itemType: reference
+          text: Update Extension Listing
+        - url: ./extensions-chromium/publish/manage-settings.md
+          itemType: reference
+          text: Manage account settings
+    # Card
+    - title: "Extensions: Microsoft Edge Addons Policies"
+      links:
+        - url: ./extensions-chromium/store-policies/csp.md
+          itemType: reference
+          text: Content Security Policy
+        - url: ./extensions-chromium/store-policies/developer-policies.md
+          itemType: reference
+          text: Microsoft Edge Addons Catalog Developer Policies
+    # Card
+    - title: Extensions for enterprise
+      links:
+        - url: ./extensions-chromium/enterprise/match-patterns.md
+          itemType: reference
+          text: Match Patterns
+        - url: ./extensions-chromium/enterprise/hosting-and-updating.md
+          itemType: reference
+          text: Hosting and Updating
+    # Card
+    - title: Test & Automation
+      links:
+        - text: Accessibility
+          itemType: reference
+          url: ./accessibility.md
+        - text: DevTools Protocol (Chromium)
+          itemType: reference
+          url: ./devtools-protocol-chromium.md
+        - text: WebDriver
+          itemType: reference
+          url: ./webdriver.md
+        - text: Puppeteer
+          itemType: reference
+          url: (to be written)
+        - text: Interop (Site compatibility)
+          itemType: reference
+          url: ./web-platform/site-impacting-changes.md
+        - text: webhint
+          itemType: reference
+          url: (nothing written, but it would be good to link out to the browser extension)
```

Comment:
You can probably link to the webhint user guide here: https://webhint.io/docs/user-guide/